Formula/avr-binutils.rb

```diff
@@ -27,8 +27,8 @@ class AvrBinutils < Formula
 
   # Support for -C in avr-size
   patch do
-    url "https://raw.githubusercontent.com/archlinux/svntogit-community/c3efadcb76f4d8b1a3784015e7c472f59dbfa7de/avr-binutils/repos/community-x86_64/avr-size.patch"
-    sha256 "7aed303887a8541feba008943d0331dc95dd90a309575f81b7a195650e4cba1e"
+    url "https://raw.githubusercontent.com/chenzhuoyu/homebrew-avr/2676e4caa70022cf7b087f7c1bb676e368343190/Patch/avr-binutils-size.patch"
```

Comment:
this must point to the patch in this repository, not in yours

Reply:
Yeah... but how do I put this patch file into this repo before... well putting this patch file into this repo? Should I split this into two PRs where the first one adds a patch file, and the second one applies the patch?

Reply:
@ladislas I reverted the changes in `avr-binutils.rb` for this PR. After this PR got merged, I will open another PR to apply the patch.